client/jetpack-connect/auth-logged-out-form.jsx

```diff
@@ -57,21 +57,23 @@ class LoggedOutForm extends Component {
 	}
 
 	renderLoginUser() {
-		const { userData, bearerToken } = this.props.jetpackConnectAuthorize;
-
-		return (
-			<WpcomLoginForm
-				log={ userData.username }
-				authorization={ 'Bearer ' + bearerToken }
-				redirectTo={ this.getRedirectAfterLoginUrl() } />
-		);
+		if ( this.props.jetpackConnectAuthorize ) {
```

Comment:
yup, because with this pr, you can get to this point without receiving a jetpackConnectAuthorize at all, redirected directly here by a link in wp-admin :)